volto/src/addons/volto-chart-builder/src/actions/index.js

```diff
@@ -12,3 +15,13 @@ export function getChartBuilderData(contentId, subRoute) {
     },
   };
 }
+
+export function getAddItemBlockData(id) {
+  return {
+    type: GET_ADD_ITEM_BLOCK_DATA,
+    request: {
```

Comment:
Include the content ID in a `payload` here, so it can be used later - see comments on the reducer